c2corg_ui/static/js/search/outingfilters.js

```diff
@@ -64,7 +63,7 @@ app.OutingFiltersController.prototype.getFilterFromPermalink = function(key) {
     }
     var dates = val.split(',');
     dates.forEach(function(date) {
-      this.dates.push(app.utils.formatDate(date));
+      this.dates.push(window.moment(date)._d);
```

Comment:
You could do it some other way? Relying on an internal property is always fragile.